spec/unit/artifactserver_spec.rb

```diff
@@ -56,6 +56,59 @@ module CookbookOmnifetch
       }
       expect(public_repo_location.lock_data).to eq(expected_data)
     end
+
+    context "when asked to install a new cookbook" do
+
+      let(:http_client) { double("Http Client") }
```

Comment:
Just a protip for the future, `instance_double` is awesome, it ensures that methods you stub exist on the class that you're doubling. In this case, though, the HTTP Client class is supposed to be DI'd (I was just too busy to follow through on this) so the HTTP Client code isn't necessarily available.